tests/backup/backup_restore_basic_test.go

```diff
@@ -593,6 +590,7 @@ var _ = Describe("{ScheduleBackupCreationSingleNS}", func() {
 				dash.VerifyFatal(err, nil, fmt.Sprintf("Verification of creating schedule backup with schedule name - %s", backupName))
```

Comment:
It is the scheduleName that we are passing to CreateScheduleBackup not backupName.